theme.go

```diff
@@ -15,7 +15,7 @@ func newTheme(themeXml xlsxTheme) *theme {
 	for _, scheme := range clrSchemes {
 		var rgbColor string
 		if scheme.SysClr != nil {
-			rgbColor = scheme.SysClr.LastClr
+			rgbColor = scheme.SysClr.LastClr // LastClr is optional, so rgbColor may be an empty string
```

Comment:
Nice comment.  I feel like we really need to make the underlying options for a colour transparent to the public interface of the library.  At the moment all of this code is pointing towards a use case where we need to know a concrete aRGB value for the colour - the only case where you need that is if you're rendering the document somewhere and this seems to be a less likely use case than generating or modifying a document.  In that case we'd be better off just preserving the stylistic information as it is.   Sadly fixing that means creating a v4 and I don't have the time or inclination to tackle it.

Reply:
Given the way we *intend* this code to be used, the right fix for this situation is to define the values for the systemColors and use those values when lastClr is omitted.